### PR TITLE
I1612 fix source 2 field

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

File: app/imports/client/react/standards/components/StandardEditForm.js [hunk 1]
```diff
@@ -106,12 +111,18 @@ export const StandardEditForm = ({ organizationId, standardId, save }) => (
...
-    <EditSourceField
-      name="source2"
-      label="Source file 2"
-      onChange={save}
-      {...{ organizationId, standardId }}
-    />
+    <Field name="shouldRenderSource2" subscription={{ value: true }}>
+      {({ input }) => (
+        <SourceFieldWrapper hidden={!input.value}>
+          <EditSourceField
```
Comment (Serhiy91) [inline app/imports/client/react/standards/components/StandardEditForm.js]: I got an error when unmounting of EditSourceField so it is hidden via css

**Comment by Serhiy91** [comment]
@Dartv I try to use FieldCondition but it causes an error.

**Comment by Dartv** [comment]
I don't see source 2 field being displayed although I added the first one

**Comment by Serhiy91** [comment]
@Dartv https://drive.google.com/file/d/1W5-jjZQvu7mkgThAMWz1dG-tacTOat8F/view

**Comment by Dartv** [comment]
@Serhiy91 doesn't work in the standard modal

**Comment by Serhiy91** [comment]
@Dartv fixed

**Pull request merged**